dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -196,13 +196,18 @@ def main(args):
         )
     client = Client(cluster)
 
-    def _worker_setup():
+    def _worker_setup(initial_pool_size=None):
         import rmm
 
-        rmm.reinitialize(pool_allocator=not args.no_rmm_pool, devices=0)
+        rmm.reinitialize(
+            pool_allocator=not args.no_rmm_pool,
+            devices=0,
+            initial_pool_size=initial_pool_size,
+        )
         cupy.cuda.set_allocator(rmm.rmm_cupy_allocator)
 
-    client.run(_worker_setup)
+    client.run(_worker_setup, 24e9)
+    client.run_on_scheduler(_worker_setup, 1e9)
```

Comment:
I think we should add a comment -- I should've done this before.  Something like:
create pool on scheduler due to occasional deserialization of cuda objects

And the same on local_cupy_transpose_sum.py

Reply:
Done.